llpc/test/shaderdb/relocatable_shaders/PipelineTess_RelocRemoveUnusedTcsOutputs.pipe

```diff
@@ -0,0 +1,80 @@
+; Test that the TCS output at location 0 is removed, and that the mapping of the TES inputs and TCS outputs match.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% \
+; RUN:         -enable-relocatable-shader-elf \
+; RUN:         -o %t.elf %gfxip %s -v | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: // LLPC location input/output mapping results (TES shader)
```

Comment:
This needs `{{^}}` to match the beginning of the line, otherwise this can match the test file itself.

Reply:
The test file is not part of the data being checked.  The "-v" option give the disassembled SPIR-V, which will include the glsl source for the shaders.  However the RUN and SHADERTEST* lines are not part of that.

Also because I used "-LABEL", it would fail if it found a second instance, so I'm not too worried about it unknowingly matching something else.

Reply:
I got somewhat paranoid after wasting a couple hours on debugging a group of test that matched fine but did the wrong thing. For example, this test does not check for `LLPC SUCCESS` so I can imagine a scenario were it did not compile anything and merely printed the input and some warning, even if this does not happen with the code today.

I'd like to have at least one beginning of line marker to be able to tell that the test is checking the actual output, without having to run the test and see what the output is to confirm that.